### PR TITLE
Update Security Dependencies

### PR DESCRIPTION
**Pull request by suvarnathekkoot-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by tferns-ch — APPROVED** [review]
Nice one, just small comments but not a big deal

File: src/main/java/uk/gov/companieshouse/accounts/association/utils/MessageType.java [hunk 1]
```diff
@@ -16,13 +16,13 @@ public enum MessageType {
 
     INVITE_MESSAGE_TYPE("associations_invite");
 
-    private final String messageType;
+    private final String messageTypeLocal;
```
Comment (kkonuganti-ch) [inline src/main/java/uk/gov/companieshouse/accounts/association/utils/MessageType.java:19]: message type local does not suit this.
can we change it to something meaningful? like value.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by kkonuganti-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**